pkg/network/common/dns.go

```diff
@@ -347,3 +361,10 @@ func removeDuplicateIPs(ips []net.IP) []net.IP {
 
 	return uniqueIPs
 }
+
+func warnSlowOperation(operation string, start time.Time) {
```

Comment:
dunno if you saw my response on slack the other day, but the thing you were looking for is `k8s.io/utils/trace`

Reply:
Note to myself: Not  outdated TODO